docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -493,6 +493,23 @@ oc patch openstackdataplanenodeset openstack --type='json' --patch='[
   }]'
 ----
 
+Note that in order to use `neutron-dhcp` with OVN for baremetal
+provisioning the `disable_ovn_dhcp_for_baremetal_ports` configuration
+option for Neutron needs to be set to `true` (it defaults to
+`false`). This configuration can be set in the `NeutronAPI` spec as
+follow:
```

Comment:
as follows

Reply:
+1 this is customer-facing docs so we need to fix such things too :)

Otherwise LGTM and adding @klgill for docs review